less/variables.less

```diff
@@ -230,18 +230,22 @@
 
 // Form states and alerts
 // -------------------------
+@warningColor:            #c09853;
```

Comment:
Isn't this a bit confusing? we now have 2 colours.

This whole section relates to colour so implicitly it is:
```css
@warningColor:            #c09853;
@warningTextColor:     @gray;
```